turbopack/crates/turbo-tasks-backend/src/backend/operation/aggregation_update.rs

```diff
@@ -203,14 +203,6 @@ impl AggregatedDataUpdate {
             for collectible in collectibles {
                 collectibles_update.push((collectible, 1));
             }
-            collectibles_update.extend(iter_many!(
-                task,
-                Collectible {
-                    collectible
-                } count => {
-                    (collectible, *count)
-                }
-            ));
```

Comment:
That's the primary bug. It is already added on line 189, so adding it here again will add it twice.